v2/internal/frontend/desktop/windows/go-webview2/pkg/edge/create_env_native.go

```diff
@@ -0,0 +1,41 @@
+//go:build !exp_gowebview2loader
+
+package edge
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+
+	"github.com/wailsapp/wails/v2/internal/frontend/desktop/windows/go-webview2/webviewloader"
+
+	"golang.org/x/sys/windows"
+)
+
+func createCoreWebView2EnvironmentWithOptions(browserExecutableFolder, userDataFolder string, environmentCompletedHandle *iCoreWebView2CreateCoreWebView2EnvironmentCompletedHandler) error {
+	browserPathPtr, err := windows.UTF16PtrFromString(browserExecutableFolder)
+	if err != nil {
+		return fmt.Errorf("Error calling UTF16PtrFromString for %s: %v", browserExecutableFolder, err)
+	}
+
+	userPathPtr, err := windows.UTF16PtrFromString(userDataFolder)
+	if err != nil {
+		return fmt.Errorf("Error calling UTF16PtrFromString for %s: %v", userDataFolder, err)
+	}
+
+	hr, err := webviewloader.CreateCoreWebView2EnvironmentWithOptions(
```

Comment:
Both the native and go versions of this call have the same package and name but have different signatures. It took me a little time to realise what was going on. Do you think changing the method name of one of the versions would be a bad idea (would that make it inconsistent)?

Reply:
My idea was to keep the method names of the original loader so people can match this to the original MS documentation to get more detailed information of the methods.

Reply:
I can change the names if you want.

Reply:
Nah, I got it after a bit and understand that rationale 👍